package.json

```diff
@@ -60,6 +59,7 @@
     "react": ">= 0.12.x"
   },
   "dependencies": {
+    "babel-eslint": "^5.0.0",
```

Comment:
Please move this to the `devDependencies` (we don't it to finish in the final bundle)